src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -36,27 +37,13 @@ public LiftSubsystem(SpeedController motor, Encoder encoder, DigitalInput lowerL
 		this.m_pidControler.setAbsoluteTolerance(pidConfig.tolerance);
 
 	}
-
-	public void periodic() {
-		double output = this.m_pidOutput;
-		if (this.m_inputSpeed != 0) {
-			this.m_isMoving = true;
-			output = this.m_inputSpeed;
-		} else if (this.m_inputSpeed == 0.0 && this.m_isMoving) {
-			this.m_pidControler.setSetpoint(this.m_encoder.pidGet());
-			this.m_isMoving = false;
-		}
-		if (this.m_encoder.get() > 5700) {
+	public void safetyChecks(double output){
+		if (this.m_encoder.get() > UPPER_ENCODER_VALUE || this.m_encoder.get() < LOWER_ENCODER_VALUE){
 			if (output > 0) {
 				output = SAFETYZONE_THROTTLE * output;
```

Comment:
What is this really doing once we exit the method and it crawls back up the stack?